deployment/config/mainnet/solana-tbr-init.json

```diff
@@ -1,10 +1,8 @@
 {
-    "owner": "11111111111111111111111111111111",
+    "owner": "8a5BXhBXGBzGRiKCCCeWH5GGGgNjDe7JyNhW65wBWoor",
     "feeRecipient": "AdAVF5KmmGmpNQhjY7FL96wZLEynD6Mx3VXJTZf2yFps",
     "admins": [
-        "11111111111111111111111111111111",
-        "11111111111111111111111111111111",
-        "11111111111111111111111111111111"
+        "tmPR6y8iFyzDuPArWGMGx21BZvEtejZ8GMSbu5Y5dGR"
```

Comment:
We probably want to remove the deployer key before merging to avoid readding it accidentally.

Reply:
Key removed.